### DISCUSSION TITLE
1.1.13

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsApp-Web-Plus#50] [Announcements]
- Critical bug fix! I had a problem with loading the modules because of WhatsApp's lazy loading method.

<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsApp-Web-Plus/releases/tag/1.1.13'>1.1.13</a>.</em>

**Comment by Blonkyblonk** [comment]
i got massage "Waiting for this message. Check your phone" when i sendin once seen mesage or got once seen message

**Reply by Schwartzblat** [reply]
Yep, my bypass is closed.
#55

**Reply by Blonkyblonk** [reply]
so there will be no more updates?

**Reply by Schwartzblat** [reply]
I don't know yet.
